src/http/cookie.cr

```diff
@@ -24,15 +24,36 @@ module HTTP
 
     def_equals_and_hash name, value, path, expires, domain, secure, http_only
 
-    def initialize(@name : String, value : String, @path : String = "/",
+    # A basic `HTTP::Cookie` can be created with a given *name* and *value*.
+    #
+    # ```crystal
+    # require "http/cookie"
+    #
+    # HTTP::Cookie.new("session", "god")
```

Comment:
This example is pretty pointless... what does it showcase actually? Using class constructors with required arguments?